_config.yml

```diff
@@ -1,19 +1,22 @@
-title: Federal Enterprise Data Resources
+title: resources.data.gov
 email: datagov@gsa.gov
 description: >-
-  Federal Enterprise Data Resources are tools, best practices, and schema standards to facilitate the adoption of open data practices across the federal government
-baseurl: "" # the subpath of your site, e.g. /blog
-url: https://resources.data.gov
+  Resources.data.gov is an online repository of policies, tools, case studies and other resources to support data governance, management, and use throughout the federal government
+baseurl: "/preview/18f/resources.data.gov/develop/" # the subpath of your site, e.g. /blog
```

Comment:
Is this necessary? Federalist will set the baseurl. The README still refers to http://localhost:4000/ which 404s with this configuration.

Reply:
Def not necessary; will remove once we're done with the 18F preview site.